fastpass/bin/fastpass

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec coursier launch org.scalameta:fastpass_2.12:1.6.0 --main scala.meta.fastpass.Fastpass -- $@
```

Comment:
Can we add a sibling `coursier` script and use that to bootstrap?

```
curl -fLo coursier https://git.io/coursier-cli &&
    chmod +x coursier &&
    ./coursier
```

```suggestion
COURSIER="$(dirname $0)/coursier"
exec $COURSIER launch fastpass --contrib -- "$@"
```

Reply:
we'd have to manually update coursier, too, when new versions are released

Reply:
I don't say, it's a bad idea, I'm just wondering

Reply:
I don't think we would have to manually update coursier, it should continue to work no matter what version of fastpass we're using

Reply:
The `coursier` slim binary should be committed into the repo

Reply:
ok, done